dockstore-webservice/src/main/java/io/dockstore/webservice/core/Tag.java

```diff
@@ -90,7 +91,7 @@ public Version createEmptyVersion() {
 
     @Override
     public Date getDate() {
-        return this.getLastBuilt();
+        return Stream.of(this.getLastBuilt(), this.getDbCreateDate()).filter(Objects::nonNull).findFirst().orElse(null);
```

Comment:
👍 Hadn't seen that technique before.

Reply:
First thing I could think of, could've sworn there was a easier way. 🤷

Reply:
Were you thinking of the Apache method? http://commons.apache.org/proper/commons-lang/javadocs/api-3.1/org/apache/commons/lang3/ObjectUtils.html#firstNonNull%28T...%29

Reply:
firstNonNull or similar does seem more readable